config/routes.rb

```diff
@@ -276,6 +276,7 @@
       get '/come_back_later' => 'come_back_later#show'
       get '/personal_key' => 'personal_key#show'
       post '/personal_key' => 'personal_key#update'
+      # Remove this after the next deploy
```

Comment:
another thought --- I think we can convert the backup codes download functionality that does something similar

Reply:
I don't quite follow?

Reply:
We do the same "download" stuff in another controller and I think if we link this client-only approach, we should consider it there, too: https://github.com/18F/identity-idp/blob/main/app/controllers/users/backup_code_setup_controller.rb#L31-L34

Reply:
ohhh got it 👍🏼